src/API/Management/Users.php

```diff
@@ -1,88 +1,262 @@
+                }
+            }
+        }
 
-        foreach ($params as $param => $value) {
-            $client->withParam($param, $value);
+        // Keep existing pagination params if passed (backwards-compat), override with non-null function param if not.
+        if (!isset($params['page']) && null !== $page) {
+            $params['page'] = abs(intval($page));
+        }
+        if (!isset($params['per_page']) && null !== $per_page) {
+            $params['per_page'] = abs(intval($per_page));
         }
 
-        return $client->call();
+        return $this->apiClient->method('get')
+            ->addPath('users')
+            ->withDictParams($params)
+            ->call();
     }
 
-    public function delete($user_id) {
+    /**
+     * Wrapper for self::search().
+     *
+     * @param array $params - Search parameters to send.
+     *
+     * @return mixed|string
+     *
+     * @throws \Exception
+     *
+     * @see self::getAll()
+     */
+    public function search($params = [])
+    {
+        return $this->getAll($params);
+    }
 
-        return $this->apiClient->delete()
-            ->users($user_id)
+    /**
+     * Delete a User by ID.
+     *
+     * @param string $user_id - User ID to delete.
+     *
+     * @return mixed|string
+     *
+     * @throws \Exception
+     */
+    public function delete($user_id)
+    {
+        return $this->apiClient->method('delete')
+            ->addPath('users', $user_id)
             ->call();
     }
 
-    public function linkAccount($user_id, $post_identities_body) {
-
-        return $this->apiClient->post()
-            ->users($user_id)
-            ->identities()
-            ->withHeader(new ContentType('application/json'))
-            ->withBody(json_encode($post_identities_body))
+    /**
+     * Link one user account to another.
+     *
+     * @param string $user_id - User ID of the primary identity where you are linking the secondary account to.
+     * @param array $data - Secondary account to link; either link_with JWT or provider, connection_id, and user_id.
+     *
+     * @return array - Array of the primary account identities.
+     *
+     * @throws \Exception
+     */
+    public function linkAccount($user_id, $data)
+    {
+        return $this->apiClient->method('post')
+            ->addPath('users', $user_id)
+            ->addPath('identities')
+            ->withBody(json_encode($data))
             ->call();
     }
 
-    public function unlinkAccount($user_id, $provider, $identity_id) {
-
-        return $this->apiClient->delete()
-            ->users($user_id)
-            ->identities($provider)
+    /**
+     * Unlink an identity from the target user.
+     *
+     * @param string $user_id - User ID to unlink.
+     * @param string $provider - Identity provider of the secondary linked account.
+     * @param string $identity_id- The unique identifier of the secondary linked account.
```

Comment:
space missing before the `-` 🔍

Reply:
hmmm... thought whitespace fixer would catch stuff like that ...